release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2752,3 +2752,19 @@ Starting with {product-title} 4.10.8, support for Google Cloud Platform Workload
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-9"]
+=== RHBA-2022:1421 - {product-title} 4.10.9 bug fix and security update
+
+Issued: 2022-04-12
+
+{product-title} release 4.10.9, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:1162[RHBA-2022:1421] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1161[RHBA-2022:1420] advisory.
```

Comment:
The link text and titles and the respective advisories that open do not match. Is this an oversight or is it because the advisories might not be published yet?